src/main/java/io/cryostat/rules/Rule.java

```diff
@@ -219,4 +230,117 @@ public String toString() {
             return getSerialKey();
         }
     }
+
+    public static Rule buildRule(MultiMap formAttributes) {
```

Comment:
This is just food for thought, I'm happy enough with this as is - what do you think about moving this into the Builder class, renaming it, and returning a Builder instance? So instead of callers doing ex. `Rule.buildRule(requestParams.getFormAttributes())`, they would do `Rule.Builder builder = Rule.Builder.from(requestParams.getFormAttributes())`. They could then still mutate the builder's state if that's needed for any reason and rely on the type safety that the Rule.Builder provides instead of trying to make the Builder do things by adding values to the JsonObject/MultiMap ahead of time. Then calling `build()` on it would give the `Rule` instance as usual.

Reply:
Good idea, this approach is more flexible. I can add that in